apps/web/components/Organization/NewsArticle/NewsArticle.tsx

```diff
@@ -39,25 +39,25 @@ export const NewsArticle: React.FC<
       </Box>
       <Box className="rs_read">
         <Text variant="intro" as="p" paddingBottom={2}>
-          {newsItem.intro}
+          {newsItem?.intro}
         </Text>
       </Box>
-      {Boolean(newsItem.image) && (
+      {Boolean(newsItem?.image) && (
         <Box
           paddingY={2}
           className={cn({
-            [styles.floatedImage]: newsItem.fullWidthImageInContent === false,
+            [styles.floatedImage]: newsItem?.fullWidthImageInContent === false,
           })}
         >
           <Image
-            {...newsItem.image}
-            url={newsItem.image.url + '?w=774&fm=webp&q=80'}
-            thumbnail={newsItem.image.url + '?w=50&fm=webp&q=80'}
+            {...newsItem?.image}
+            url={newsItem?.image?.url + '?w=774&fm=webp&q=80'}
+            thumbnail={newsItem?.image?.url + '?w=50&fm=webp&q=80'}
```

Comment:
Here we are potentially doing undefined + "?w= 50&fm=webp&q=80" which will result in "undefined?w= 50&fm=webp&q=80"